docs/endpoints/multi_user.rst

```diff
@@ -0,0 +1,909 @@
+the filename communicates the same idea!)  In particular, note that the mapping of
+identities from Globus Auth that are passed with each "start UEP" request is a key part
+of the MEP setup and security.
+
+.. warning::
+
+   At the Globus Compute web service, MEPs are, by default, open.  Any (Globus Auth
+   authenticated) user may submit tasks to them.  The intended and primary means of
+   access control is the identity mapping configuration.  For more information, see
+   `An Open Endpoint`_
+
+.. code-block:: json
+   :caption: The default example identity mapping configuration; technically functional
+       but pragmatically useless
+
+   [
+     {
+       "comment": "For more examples, see: https://docs.globus.org/globus-connect-server/v5.4/identity-mapping-guide/",
+       "DATA_TYPE": "external_identity_mapping#1.0.0",
+       "command": ["/bin/false", "--some", "flag", "-a", "-b", "-c"]
+     },
+     {
+       "comment": "For more examples, see: https://docs.globus.org/globus-connect-server/v5.4/identity-mapping-guide/",
+       "DATA_TYPE": "expression_identity_mapping#1.0.0",
+       "mappings": [
+         {
+           "source": "{username}",
+           "match": "(.*)@example.com",
+           "output": "{0}"
+         }
+       ]
+     }
+   ]
+
+The file is a JSON list of identity mapping configurations that will be tried in order.
+By implementation within the MEP code base, the first configuration to return a match
+"wins."  In this example, the first configuration is a call out to an external tool,
+as specified by the ``external_identity_mapping#1.0.0`` DATA_TYPE.  The command is a
+list of arguments, with the first element as the actual executable.  In this case,
+the flags are strictly illustrative, as ``/bin/false`` always returns with a non-zero
+exit code and so will be ignored by the |globus-identity-mapping|_ logic.
+
+The second configuration in this example is an ``expression_identity_mapping#1.0.0``,
+which means it uses a subset of regular expression syntax to search for a suitable
+POSIX username.  This configuration searches the ``username`` field from the passed
+identity (or identities, if the user has multiple linked Globus Auth identities) for
+a value that ends in ``@example.com``.  The library appends the ``^`` and ``$`` anchors
+to the regex before searching, so the actual regular expression used would be
+``^(.*)@example.com$``.  Finally, if a match is found, the first saved group is the
+output (i.e., ``{0}``).  If the ``username`` field contained ``mickey97@example.com``,
+then this configuration would return ``mickey97``, and the MEP would then use
+`getpwnam(3)`_ to look up ``mickey97``.  But if the username field(s) did not end with
+``@example.com``, then it would not match and the start-UEP request would fail.
+
+For a much more thorough dive into the identity mapping configurations, please consult
+the `Identity Mapping documentation`_.
+
+
+``user_config_template.yaml``
```

Comment:
I'm remembering a mental note to make this file use `.yaml.j2` extension for clarity. Thoughts on getting that in before release?

Reply:
Uh ... not before release.  I thought about this as well.  Fancy a creating a ticket?

Reply:
Already done: https://app.shortcut.com/globus/story/33651/use-j2-extension-for-user-config-template.